### ISSUE TITLE
Pulse Preview Bug - Null Dates Replaced with Today

### ISSUE THREAD

**Issue by lindsay-stevens** [metabase/metabase#9426]
## Environment

- Your browser and the version: Chrome 72
- Your operating system: Windows 10, host CentOS 7
- Your databases: Postgres 10 for app and sources
- Metabase version: Community v0.28.6 and Enterprise v1.0.6
- Metabase hosting environment: CentOS 7 systemd service
- Metabase internal database: Postgres 10

## Bug

- Create question with a nullable date column
- Create a pulse based on that question

Expected:

- Question shows null when date is null
- Pulse preview shows null when date is null
- Pulse attachment data shows null when date is null

Actual:

- Pulse preview shows today's date instead of null

## Evidence

1 - Example SQL for the Sample Dataset to reproduce bug.

```sql
SELECT 
  BIRTH_DATE,
  NAME,
  ID,
  CASE
    WHEN row_number() over () % 2 = 0
      THEN NULL
    ELSE BIRTH_DATE
  END AS date_nulls,
  CASE
    WHEN row_number() over () % 3 = 0
      THEN NULL
    ELSE NAME
  END AS text_nulls,
  CASE
    WHEN row_number() over () % 4 = 0
      THEN NULL
    ELSE ID
  END AS int_nulls
FROM PEOPLE
```

2 - SQL Question results - all fine for a date column, text column, and int column.

![question rendering](https://user-images.githubusercontent.com/6345269/52835235-bcaf1b00-3139-11e9-9b91-006f384ebb00.PNG)

3 - Pulse preview (same for Pulse UI and email) - note that NULLs are replaced with today's date.

![pulse rendering](https://user-images.githubusercontent.com/6345269/52835247-cd5f9100-3139-11e9-9f99-51bbf1cd90d3.PNG)

4 - Pulse CSV attachment - all fine.

![pulse attachment data](https://user-images.githubusercontent.com/6345269/52835264-dfd9ca80-3139-11e9-9eb3-4f6d0b0cf4a2.PNG)

**Comment by adegeofroy21** [comment]
I came here to create the same issue, and I can verify the problem. My scenario is nearly the same, except I am using MongoDB as the database. I see the exact same behavior: the body of the email includes today's date in place of null values.

**Comment by flamber** [comment]
This was fixed in 0.35.4 - between 0.34.0 and 0.35.3 it would create the `temporal` stacktrace #11665
